wes/wes_output_API.json

```diff
@@ -3,98 +3,98 @@
+            "long_description": "This file reports the MHC class I and II results.  The class I alleles are derived from the OptiType results and the class II alleles come from the xHLA results. ",
             "filter_group": "HLA",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/neoantigen/{run id}/combined/{run id}.filtered.tsv",
             "short_description": "neaontigen: list of predicted neoantigens",
-            "long_description": "neaontigen: list of predicted neoantigens",
+            "long_description": "The combined MHC class I and II prediected neoantigens using the pVACseq software.  The column definitions are given here (ref: https://pvactools.readthedocs.io/en/latest/pvacseq/output_files.html)",
             "filter_group": "neoantigen",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/purity/{run id}/{run id}.optimalpurityvalue.txt",
             "short_description": "tumor purity: tumor purity analysis results",
-            "long_description": "tumor purity: tumor purity analysis results",
+            "long_description": "Tumor purity calculations using the FACETS software.",
             "filter_group": "purity",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/clonality/{run id}/{run id}_pyclone.tsv",
             "short_description": "tumor clonality: tumor clonality analysis results",
-            "long_description": "tumor clonality: tumor clonality analysis results",
+            "long_description": "Tumor clonality results using PyClone software.",
             "filter_group": "clonality",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/copynumber/{run id}/{run id}_cnvcalls.txt",
             "short_description": "copynumber: copynumber analysis results",
-            "long_description": "copynumber: copynumber analysis results",
+            "long_description": "Copy number variation analysis results using Sentieon CNV algorithm",
             "filter_group": "copynumber",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/copynumber/{run id}/{run id}_cnvcalls.txt.tn.tsv",
             "short_description": "copynumber: copynumber analysis results",
-            "long_description": "copynumber: copynumber analysis results",
+            "long_description": "Segmented copy number variation file using Sentieon CNV algorithm",
             "filter_group": "copynumber",
             "file_purpose": "Analysis file"
         },
         {
             "file_path_template": "analysis/report.zip",
             "short_description": "wes report: wes summary html report",
-            "long_description": "wes report: wes summary html report",
+            "long_description": "WES summary report",
```

Comment:
Can you elaborate on this further, so the `long description` encapsulates an overview of sections covered in the report in greater detail than the short description?